integreat_cms/cms/models/media/media_file.py

```diff
@@ -97,20 +97,35 @@ def upload_path_thumbnail(instance, filename):
     return path
 
 
+
+def file_size_limit(value): 
+
+        limit = 3 * 1024 * 1024
+        if value.size > limit:
```

Comment:
```suggestion
        if value.size > settings.MEDIA_MAX_UPLOAD_SIZE:
```
I think it makes sense to convert this hard-coded value to a setting in the [settings.py](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/core/settings.py), e.g.:
```
#: The maximum size of media files in bytes
MEDIA_MAX_UPLOAD_SIZE = int(
    os.environ.get("INTEGREAT_CMS_MEDIA_MAX_UPLOAD_SIZE", 3 * 1024 * 1024)
)
```